### PR TITLE
Extract original errors

### PR DESCRIPTION
**Pull request by dmitriyslaym**
#29

### CODE REVIEW

File: packages/workspace/src/helpers/const.ts [hunk 1]
```diff
@@ -6,9 +6,23 @@ export const configNotLoadedError = 'Workspace configuration can not be loaded';
 
 export const createWorkspaceWrongRequestError = 'createWorkspace has been called with invalid token';
 
-export const bootstrapServiceError = 'Something is wrong while bootstrapping a service';
+export const getLoadingServiceError = (error: Error, serviceMame: string) =>
```
Comment (MikeSquall) [inline packages/workspace/src/helpers/const.ts]:
```suggestion
export const getLoadingServiceError = (error: Error, serviceName: string) =>
```

Comment (MikeSquall) [reply packages/workspace/src/helpers/const.ts]: You have a typo here and below the same for almost every messages

File: packages/workspace/src/helpers/utils.ts [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
 import { API } from '..';
 import * as INTERNAL_TYPES from './types';
-import { bootstrapComponentError, bootstrapServiceError } from './const';
+import {
+  getLoadingServiceError,
+  getLoadingComponentError,
+  getBootstrapServiceError,
+  getInitComponentError,
+  getBootstrapComponentError,
+} from './const';
```
Comment (MikeSquall) [inline packages/workspace/src/helpers/utils.ts:10]: This shouldn't be on some `errors.ts` file just to be more clear ?

File: packages/workspace/src/helpers/utils.ts [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
 
 export const getConfigurationService = (token: string): ConfigurationService<API.WorkspaceConfig> =>
   new ConfigurationServiceHttp(token);
 
+export const dynamicImport = (path: string) => import(path).then((module) => module.default);
```
Comment (MikeSquall) [inline packages/workspace/src/helpers/utils.ts:15]: Is this implicitly returning a Promise ?

File: packages/workspace/src/helpers/utils.ts (hunk 2 above)
Comment (dmitriyslaym) [reply packages/workspace/src/helpers/utils.ts:10]: I was thinking about it, but errors are just conts - seems enough to me

File: packages/workspace/src/helpers/const.ts (hunk 1 above)
Comment (dmitriyslaym) [reply packages/workspace/src/helpers/const.ts]: thanks!!

File: packages/workspace/src/helpers/utils.ts (hunk 3 above)
Comment (dmitriyslaym) [reply packages/workspace/src/helpers/utils.ts:15]: explicitly - no need to specify it

**Review by MikeSquall — APPROVED** [review]

**Review by VictoriaBordian — APPROVED** [review]
QA passed

**Pull request merged**